metal/datasource_metal_spot_market_price.go

```diff
@@ -29,26 +33,45 @@ func dataSourceSpotMarketPrice() *schema.Resource {
 
 func dataSourceMetalSpotMarketPriceRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*packngo.Client)
-
+	sms := client.SpotMarket.(*packngo.SpotMarketServiceOp)
```

Comment:
https://github.com/packethost/packngo/pull/264 would avoid this, but it works until we release a new packngo.